src/components/article.js

```diff
@@ -4,7 +4,15 @@ import FlatButton from 'material-ui/FlatButton';
 
 
 const Article = (props) => {
-  const image = props.data.multimedia[0] ? props.data.multimedia[4].url : null;
+  let image = null;
+  if (props.data.multimedia.length > 0) {
```

Comment:
You should add a test passing in 0, 3, 4, and 5 length multimedia arrays to ensure this is behaving as you expect.